solutions/observability/observability-ai-assistant.md

```diff
@@ -202,6 +202,20 @@ After creating the pipeline, complete the following steps:
 
     Ask something to the AI Assistant related with the indexed data.
 
+### Add user-specific system prompts
+
+User-specific prompts customize how the AI assistant responds by appending personalized instructions to all of your system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.
```

Comment:
Perhaps we should make it clear that user instructions are appended to the built-in prompts.


```suggestion
User-specific prompts customize how the AI assistant responds by appending personalized instructions to built-in system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.
```